test/integration/test_controlplane.go

```diff
@@ -392,6 +397,186 @@ func TestControlPlaneEssentials(t *testing.T) {
 	)
 }
 
+func TestControlPlaneWatchNamespaces(t *testing.T) {
+	t.Parallel()
+	namespace, cleaner := helpers.SetupTestEnv(t, GetCtx(), GetEnv())
+
+	dataplaneClient := GetClients().OperatorClient.GatewayOperatorV1beta1().DataPlanes(namespace.Name)
+	controlplaneClient := GetClients().OperatorClient.GatewayOperatorV1beta1().ControlPlanes(namespace.Name)
+
+	dp := &operatorv1beta1.DataPlane{
```

Comment:
Can we use the `DataPlane` builder in https://github.com/Kong/gateway-operator/blob/cp-mirror/controller/pkg/builder/builder.go#L17-L32?

Reply:
Done in a2337b219081303cd49b9d1d57ca233cf48d677d